graql/reasoner/concept-inequality.feature

```diff
@@ -572,15 +569,12 @@ Feature: Concept Inequality Resolution
     Then for graql query
       """
       match
+        $value isa! retailer;
+        $unwantedValue isa! retailer;
         $x has $value;
         $y has $unwantedValue;
-        not { $value is $unwantedValue; };
         $unwantedValue "Ocado";
-        $value isa! $type;
-        $unwantedValue isa! $type;
-        $type type base-attribute;
-        not { $type is $unwantedType; };
-        $unwantedType type string-attribute;
+        not { $value is $unwantedValue; };
       get $x, $value, $type;
```

Comment:
`$type` is no longer present

Reply:
interesting - this should probably be an issue in Graql

Reply:
addressed